src/main/java/com/ukubuka/core/reader/UkubukaReader.java

```diff
@@ -18,6 +18,10 @@
 import com.ukubuka.core.model.SupportedSource;
 import com.ukubuka.core.utilities.Constants;
 
+import javax.xml.stream.XMLInputFactory;
+import javax.xml.stream.XMLStreamException;
```

Comment:
Create `Spring Beans` for all the final thread-safe instances.

Reply:
Done.